modules/text/src/precomp.hpp

```diff
@@ -48,6 +48,9 @@
 #include "text_config.hpp"
 
 #ifdef HAVE_TESSERACT
+#if !defined(USE_STD_NAMESPACE)
+#define USE_STD_NAMESPACE
```

Comment:
I think problem is explained [here](https://github.com/tesseract-ocr/tesseract/pull/1082)